bin/generate.dart

```diff
@@ -45,7 +45,7 @@ class DiagramGenerator {
   /// The path to the dart program to be run for generating the diagram.
   static final String generatorDir = path.join(
     projectDir,
-    'packages',
```

Comment:
shouldn't this follow the folder structure?

In my pr I updated the test instead